cdap-docs/examples-manual/source/examples/user-profiles.rst

```diff
@@ -0,0 +1,245 @@
+
+- Build the example (as described `below <#building-an-example-application>`__).
+- Start CDAP, deploy and start the application and its component as described below in 
+  `Running CDAP Applications`_\ .
+  Make sure you start the Flow and Service as described below.
+- Once the application has been deployed and started, you can `run the example. <#running-the-example>`__
+- You should observe errors as described.
+
+Re-build the Application with Column-level Conflict Detection
+-------------------------------------------------------------
+
+- Stop the Application's Flow and Service (as described `below <#stopping-the-application>`__).
+- Delete the ``profiles`` Dataset, either through the CDAP Command Line Interface or
+  by making a ``curl`` call::
+
+    curl -w '\n' -v localhost:10000/v2/data/datasets/profiles -XDELETE
+
+- Now, rebuild the application, setting the ``tableProperties`` back to its original value, ``ConflictDetection.COLUMN``.
+- Re-deploy and re-run the application. You should not see any errors now in the log.    
+
+.. **Note:** A version of ``curl`` that works with Windows is included in the CDAP Standalone
+.. SDK in ``libexec\bin\curl.exe``
+
+
+Running CDAP Applications
+============================================
+
+.. include:: /../../developers-manual/build/_includes/building-apps-versioned.rst
+    :start-line: 9
+
+Running the Example
+===================
+
+Deleting an Existing Dataset
+----------------------------
+
+If it has been created from an earlier run of the example, delete the ``profiles``
+Dataset, either through the CDAP Command Line Interface or by making a ``curl`` call::
+
+  curl -w '\n' -v localhost:10000/v2/data/datasets/profiles -XDELETE
+
+
+Starting the Service and the Flow
+---------------------------------
+
+Once the application is deployed:
+
+- Click on ``UserProfiles`` in the Overview page of the CDAP Console to get to the
+  Application detail page, click:
+  
+  - ``ActivityFlow`` in the *Flow* pane to get to the Flow detail page, then click the *Start* button; or
+  - ``UserProfileService`` in the *Service* pane to get to the Service detail page, then click the *Start* button; and
+  
+- From the Standalone CDAP SDK directory, use the Command Line Interface:
+
+  .. list-table::
+    :widths: 20 80
+    :stub-columns: 1
+
+    * - On Linux:
+      - ``$ ./bin/cdap-cli.sh start flow UserProfiles.ActivityFlow``
+    * - 
+      - ``$ ./bin/cdap-cli.sh start service UserProfiles.UserProfileService``
+      
+..    * - On Windows:
+..      - ``> bin\cdap-cli.bat start flow UserProfiles.ActivityFlow``    
+..    * - 
+..      - ``> bin\cdap-cli.bat start service UserProfiles.UserProfileService``    
+
+Populate the ``profiles`` Table
+-------------------------------
+
+Populate the ``profiles`` tables with users using a script. From the example's directory, use:
+
+  .. list-table::
+    :widths: 20 80
+    :stub-columns: 1
+
+    * - On Linux:
+      - ``$ ./bin/add-users.sh``
```

Comment:
Do you want mention somewhere that or why there is no instructions for windows? If so, the you don't need to repeat "On Linux" all the time.